doc/user-guide/cli.md

```diff
@@ -207,13 +207,12 @@ ploomber-cloud deploy
 The command-line interface will automatically read and encrypt your secrets and include them in the deployment.
 For security reasons, your `.env` file is replaced with an empty file at runtime. Ploomber only stores your encrypted secrets.
 
+To learn how to read your secrets from within your application, see [Reading secrets.](../user-guide/secrets.md#reading-secrets)
+
 ```{note}
-Ensure that you specify all the required secrets in the `.env` file whenever deploying or re-deploying an application. Secrets will not be carried over from the previous deployment.
+We generally advise users to define secrets through an `.env` file. In the case this isn't possible, we've created an [alternative solution.](../user-guide/github.md#secrets)
```

Comment:
I'd phrase this differently. something like "by default, you can store secrets in an .env file, but you can also read from environment variables"

Reply:
and also add an admonition at the top saying that users *should not* upload an .env file to github and link to the solution

Reply:
[Rephrased](https://github.com/ploomber/doc/pull/212/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R217) and added important admonition to [warn users not to upload .env file](https://github.com/ploomber/doc/pull/212/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R194-R196)